sdk/core/azure-core-http-netty/src/main/java/com/azure/core/http/netty/NettyAsyncHttpClientBuilder.java

```diff
@@ -236,7 +236,7 @@ public NettyAsyncHttpClientBuilder connectionProvider(ConnectionProvider connect
         if (connectionProvider != null) {
             LOGGER.verbose("Setting ConnectionProvider for the Reactor Netty HttpClient. Please be aware of the "
                 + "differences in runtime behavior when creating a default Reactor Netty HttpClient vs an HttpClient"
-                + "with a specified ConnectionProvider. For more details see " 
+                + "with a specified ConnectionProvider. For more details see "
```

Comment:
The JavaDoc for this builder should be updated to make it clear that the client built from this can support sync and async sending. See https://github.com/Azure/azure-sdk-for-java/pull/32840#discussion_r1067516563